internal/event/event.go

```diff
@@ -13,7 +13,7 @@ import (
 
 type Event struct {
 	Time     time.Time
-	SourceId int64 `json:"source_id"`
+	SourceId int64
```

Comment:
IIRC you could still set this as `{"SourceId": 42}` this way, so you probably want to explicitly specify `json:"-"` now.